src/Components/SearchInput.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import { withRouter } from "react-router";
+import styled from "styled-components";
+
+import searchIcon from 'assets/svgs/search.svg';
+
+const SearchInputForm = styled.form`
+    border: 1px solid #ccc;
+    display: flex;
+    align-items: center;
+
+    input{
+        background: transparent none repeat scroll 0 0;
+        padding: 9px 50px 9px 10px;
+        border: medium none;
+        height: 20px;
+        color: #666666;
+        outline: medium none;
+
+        @media (max-width: 600px){
+            height: 30px;
+            width: 80%;
+            margin: 1rem 0;
+        }
+    }
+
+    button{
+        background: url(${searchIcon}) no-repeat;
+        background-size: 1rem;
+        width: 1rem;
+        height: 1rem;
+        border: none;
+        margin-right: 10px;
+        cursor: pointer;
+    }
```

Comment:
I believe these should be extracted to their own styled components, so you can reuse them later.